addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -68,6 +68,29 @@ MockServer.include({
         ];
         this._widget.call('bus_service', 'trigger', 'notification', [notifConfirmFold]);
     },
+    /**
+     * Simulate the 'channel_seen' route of of mail.channel
+     * In particular sends a notification on the bus
+     *
+     * @private
+     */
+     _mockChannelSeen(args) {
+        const { last_message_id } = args.kwargs;
+        const channel_id = args.args[0][0];
+        // Should use something of the form session.partner_id or env.messaging.currentPartner
+        // This is a limitation of the architecture of JS tests
```

Comment:
I think this limitation is lifted by @seb-odoo's PR on mock server changes, if I am right?

Reply:
Yes I just stored a param currentPartnerId in the mock server to avoid using hardcoded ids for current partner.

But here it doesn't change anything for bus notifications, the partner id is just used by the server to know to whom send it on the bus, but here we are already working with the `bus_service` of the target user so the value is irrelevant.